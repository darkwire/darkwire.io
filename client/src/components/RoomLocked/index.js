import React, { Component } from 'react'

export default class RoomLocked extends Component {
  render() {
    return (
      <div>
        This room is locked.
      </div>
    )
  }
}
