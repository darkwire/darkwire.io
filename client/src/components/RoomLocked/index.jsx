const RoomLocked = ({ modalContent }) => {
  return <div>{modalContent}</div>;
};

export default RoomLocked;
