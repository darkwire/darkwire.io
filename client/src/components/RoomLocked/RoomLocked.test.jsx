import React from 'react';
import { render } from '@testing-library/react';
import RoomLocked from '.';
import { test, expect } from 'vitest';

test('RoomLockedÂ component should display', () => {
  const { asFragment } = render(<RoomLocked modalContent={'test'} />);
  expect(asFragment()).toMatchSnapshot();
});
