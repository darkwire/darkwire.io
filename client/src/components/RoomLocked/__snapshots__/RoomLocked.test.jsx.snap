// Vitest Snapshot v1

exports[`RoomLockedÂ component should display 1`] = `
<DocumentFragment>
  <div>
    test
  </div>
</DocumentFragment>
`;
